flink-libraries/flink-ml/src/test/scala/org/apache/flink/ml/math/distributed/BlockMatrixSuite.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.math.distributed
+
+import org.apache.flink.api.scala._
+import org.apache.flink.ml.math.SparseMatrix
+import org.apache.flink.test.util.FlinkTestBase
+import org.scalatest.{FlatSpec, GivenWhenThen, Matchers}
+
+class BlockMatrixSuite
+    extends FlatSpec
+    with Matchers
+    with GivenWhenThen
+    with FlinkTestBase {
+
+  val env = ExecutionEnvironment.getExecutionEnvironment
```

Comment:
I found Travis-CI is failed because of this line. We cannot reuse `ExecutionEnvironment` due to order of class initialization. It seems related to FLINK-3994. Please move initialization of `ExecutionEnvironment` and `DistributedRowMatrix` into each test case.